### ISSUE TITLE
Websocket connection closed when joining a voice channel

### ISSUE THREAD

**Issue by abdieg** [Rapptz/discord.py#1143]
Hi,
Today (March 15th) at about 12am GMT-6, my bot was working correctly. Now I found in the morning the following error displayed when I try to connect to a voice channel:

```
Bot unido al canal de voz: Sound & Emotion
Fatal error on SSL transport
protocol: <asyncio.sslproto.SSLProtocol object at 0x7fa28bb01e80>
transport: <_SelectorSocketTransport closing fd=9 read=idle write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.6/asyncio/sslproto.py", line 638, in _process_write_backlog
    ssldata, offset = self._sslpipe.feed_appdata(data, offset)
AttributeError: 'NoneType' object has no attribute 'feed_appdata'
Ignoring exception in on_command_error
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/gateway.py", line 676, in poll_event
    msg = yield from asyncio.wait_for(self.recv(), timeout=30.0, loop=self.loop)
  File "/usr/lib/python3.6/asyncio/tasks.py", line 358, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.6/dist-packages/websockets/protocol.py", line 319, in recv
    raise ConnectionClosed(self.close_code, self.close_reason)
websockets.exceptions.ConnectionClosed: WebSocket connection is closed: code = 4006, reason = Session is no longer valid..

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 50, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "PerronBot.py", line 490, in join
    VC = await BOT.join_voice_channel(voice_channel)
  File "/usr/local/lib/python3.6/dist-packages/discord/client.py", line 3211, in join_voice_channel
    yield from voice.connect()
  File "/usr/local/lib/python3.6/dist-packages/discord/voice_client.py", line 258, in connect
    self.ws = yield from DiscordVoiceWebSocket.from_client(self)
  File "/usr/local/lib/python3.6/dist-packages/discord/gateway.py", line 591, in from_client
    yield from asyncio.wait_for(ws.poll_event(), timeout=60, loop=client.loop)
  File "/usr/lib/python3.6/asyncio/tasks.py", line 358, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.6/dist-packages/discord/gateway.py", line 679, in poll_event
    raise ConnectionClosed(e) from e
discord.errors.ConnectionClosed: WebSocket connection is closed: code = 4006, reason = Session is no longer valid..

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "PerronBot.py", line 718, in on_command_error
    raise error  #No borrar, para que todos los errores sigan siendo mostrados en consola
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/bot.py", line 846, in process_commands
    yield from command.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 374, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 54, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: ConnectionClosed: WebSocket connection is closed: code = 4006, reason = Session is no longer valid..
```

I attach my code, but I assure you that my code was working fine at the mentioned hour:

```python
@BOT.command(pass_context=True)
@commands.cooldown(1, 30, commands.BucketType.server)
async def join(ctx):
    global VC
    VC = None
    server = ctx.message.server
    author = ctx.message.author
    voice_channel = author.voice_channel
    print("Bot unido al canal de voz: " + str(voice_channel))
    if voice_channel is None:
        await BOT.say("Tienes que estar dentro de un canal de voz para agregar al bot")
        return

    if BOT.is_voice_connected(server) is True:
        await BOT.say("El Bot Perron ya está dentro de un canal de voz")
    else:
        VC = await BOT.join_voice_channel(voice_channel)
        await BOT.say("Bot Perron se ha unido a " + voice_channel.name)
```
Current Discord.py Version: 0.16.12 | Current Python Version: 3.6.3

Is there any problem with the Discord API? This is the first time that I see this error.

Also, for some reason, now the bot is duplicating the messages it send: This didn't happen before:

![capture](https://user-images.githubusercontent.com/13859442/37481025-4828edc6-2846-11e8-927d-69fc997ac13f.PNG)

Thanks in advance

**Comment by abdieg** [comment]
Someone just gave me the idea that (for some reaon) the Python instance of my bot could be duplicated in my server. So I ran HTOP and I found that indeed, my bot was duplicated. I killed all the python processes and executed the bot again and it worked again as expected. 
I apologyze for any inconvenience.

**Comment by Harmon758** [comment]
> Fatal error on SSL transport
protocol: <asyncio.sslproto.SSLProtocol object at 0x7fa28bb01e80>
transport: <_SelectorSocketTransport closing fd=9 read=idle write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.6/asyncio/sslproto.py", line 638, in _process_write_backlog
    ssldata, offset = self._sslpipe.feed_appdata(data, offset)
AttributeError: 'NoneType' object has no attribute 'feed_appdata'

Note, this was a separate [issue with asyncio](https://bugs.python.org/issue33037) and should be fixed in Python 3.6.5 now.
